File: ios/FantaSensi/Auth/AuthConfig.swift
import Foundation

enum AuthConfig {
    private static let nameEnc = "bUD0hgToHEL27IpUAhIC6SYdjqdADSTY5xcvs9W0Hc4="
    private static let ownerEnc = "oquS3Y7Lv4qX29+kPK3xGHKwJZ3qmtTYvCH2maFk/V8="
    private static let versionEnc = "Wc3s0/HaVH/qDozGkA2XW49s4JELsw9+/wM51UjYJyI="
    private static let urlEnc = "CNf4jBG7dxFIyaGhHOi4Ritk10tt2xziNxU2QCoyiLdyJdvo2MHCWXrAv1JBHUeH"

    static var name: String { Obf.d(nameEnc) }
    static var ownerId: String { Obf.d(ownerEnc) }
    static var version: String { Obf.d(versionEnc) }
    static var url: String { Obf.d(urlEnc) }
}

File: android/app/src/main/java/com/fantasensi/mobile/auth/AuthConfig.kt
package com.fantasensi.mobile.auth

object AuthConfig {
    private val nameEnc = "QGC5OR3t1dZ8yHGJhHAOIuyeqEkE61PifNHItVwk38Q="
    private val ownerEnc = "hOXXBLGvLW6mrjX/m1ARE2x/gddQpzoefctErBYrxd8="
    private val versionEnc = "Wc3s0/HaVH/qDozGkA2XW49s4JELsw9+/wM51UjYJyI="
    private val urlEnc = "CNf4jBG7dxFIyaGhHOi4Ritk10tt2xziNxU2QCoyiLdyJdvo2MHCWXrAv1JBHUeH"

    val name: String get() = Obf.d(nameEnc)
    val ownerId: String get() = Obf.d(ownerEnc)
    val version: String get() = Obf.d(versionEnc)
    val url: String get() = Obf.d(urlEnc)
}
